include/wx/android/setup.h

```diff
@@ -462,6 +462,15 @@
 // Recommended setting: 1 (but may be safely disabled if you don't use it)
 #define wxUSE_SECRETSTORE   1
 
+// Allow the use of the OS built-in spellchecker on wxTextCtrl (multiline
+// only). Currently wxGTK, wxMSW and wxOSX implemented.
```

Comment:
So, should this be enabled for Android?

Reply:
No, and I honestly can't remember why these lines are included in the Android, wxUniv, Motif setup.h.

Reply:
Probably because it is there in `include/wx/setup_inc.h`?

Reply:
I see, it is automatically added by `build/update-setup-h`.  I have changed the default setting to be conditional on __WXGTK3__ and __WXMSW__.